warehouse/static/js/warehouse/controllers/github_repo_stats_controller.js

```diff
@@ -15,28 +15,45 @@ import { Controller } from "@hotwired/stimulus";
 
 export default class extends Controller {
   static outlets = ["github-repo-info"];
-  static values = { url: String };
+  static values = {
+    url: String,
+    issueUrl: String,
+  };
 
   connect() {
     this.load();
   }
 
   load() {
-    fetch(this.urlValue, {
+    const fetchParams = {
       method: "GET",
       mode: "cors",
-    })
-      .then((response) => response.ok === true ? response.json() : null)
-      .then((data) => {
+    };
+    const fetchRepoData = fetch(this.urlValue, fetchParams).then((response) =>
+      response.ok === true ? response.json() : null
+    );
+
+    const fetchIssueData = fetch(this.issueUrlValue, fetchParams).then(
+      (response) => (response.ok === true ? response.json() : null)
+    );
+
+    const allData = Promise.all([fetchRepoData, fetchIssueData]);
+
+    allData
+      .then((res) => {
         const stats = {
-          bugs: data.open_issues_count,
-          bugs_url: data.html_url + "/issues",
-          followers: data.stargazers_count,
-          followers_url: data.html_url + "/stargazers",
-          forks: data.forks_count,
-          forks_url: data.html_url + "/network/members",
+          issues_url: res[0].html_url + "/issues",
+          PRs_url: res[0].html_url + "/pulls",
+          followers: res[0].stargazers_count,
+          followers_url: res[0].html_url + "/stargazers",
+          forks: res[0].forks_count,
+          forks_url: res[0].html_url + "/network/members",
+          issues: res[1].total_count,
+          PRs: res[0].open_issues_count - res[1].total_count,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Now that both API calls are performed concurrently in the Promise, what happens if either of them return `null`? Will calls to `res[N].some_property` fail?

Reply:
Based on the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) if one fails the whole expression throws and we would be in the catch(). The catch() does nothing, which was the behavior prior to this PR. However, if there is interest in changing this it could be possible to use [promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) and handle the errors separately.

Reply:
No further change needed, thanks for clarifying the behaviors.